src/main/resources/db/changelog/changes/init.sql

```diff
@@ -77,4 +77,7 @@ COMMENT ON TABLE public.holders_credential IS 'This table will store holders cre
 COMMENT ON COLUMN public.holders_credential.is_stored IS 'true is VC is stored using store VC api(Not issued by MIW)';
 
 --changeset nitin:2
-ALTER TABLE public.wallet_key ADD key_id varchar(255) NULL;
\ No newline at end of file
+ALTER TABLE public.wallet_key ADD key_id varchar(255) NULL;
+
+--changeset pmanaras:3
+ALTER TABLE public.wallet ADD key_storage_type VARCHAR(255) NOT NULL;
```

Comment:
We need to set a default value to DB for existing wallets

Reply:
you are right, if the rest of the impl is fine (removed the uncessary imports and vars) i'll write a migration script